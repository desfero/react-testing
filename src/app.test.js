import React from 'react';
import {shallow} from 'enzyme';
import {App} from './app';

describe('App', () => {
  it('should render the compoent', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
