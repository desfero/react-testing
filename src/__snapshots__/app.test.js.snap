// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the compoent 1`] = `
<main>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</main>
`;
